### PR TITLE
Address EKR's 'first look' comments

### PR DESCRIPTION
**Pull request by selfissued**

### CODE REVIEW

File: draft-ietf-oauth-device-flow.xml [hunk 1]
```diff
@@ -636,6 +636,14 @@
           from. Thus, signals from the approving user's session and device are
           not relevant to the trustworthiness of the client device.
           </t>
+	  <t>
+	    Note that if an authorization server used with this flow is malicious,
+	    then it could man-in-the middle the backchannel flow to another authorization server.
+	    For this to be possible, the device manufacturer must either be the attacker
```
Comment (WilliamDenniss) [inline draft-ietf-oauth-device-flow.xml]: "for this attack to be possible" repeated below

Comment (WilliamDenniss) [inline draft-ietf-oauth-device-flow.xml]: Regarding "the device manufacturer must either be the attacker" are you documenting a possibility where an honest app is running on an evil operating system, which is doing the MITM?

File: draft-ietf-oauth-device-flow.xml [hunk 2]
```diff
@@ -646,14 +654,17 @@
           inform the user that they are authorizing a device during the user
           interaction step (see <xref target="UserInteraction"/>), and to
           confirm that the device is in their possession.
+	  The authorization server SHOULD display information about the device
+	  so that the person would notice if a software client
```
Comment (WilliamDenniss) [inline draft-ietf-oauth-device-flow.xml]: would -> could?

The last IESG review I went through, my doc was criticized for putting too much faith in the user. I think with this SHOULD the user has the ability to notice, but may miss the signal.

File: draft-ietf-oauth-device-flow.xml [hunk 3]
```diff
@@ -636,6 +636,14 @@
...
           not relevant to the trustworthiness of the client device.
           </t>
+	  <t>
+	    Note that if an authorization server used with this flow is malicious,
+	    then it could man-in-the middle the backchannel flow to another authorization server.
+	    For this to be possible, the device manufacturer must either be the attacker
+	    or be using an authorization server that is controlled by an attacker
+	    for this attack to be possible. In part,
+	    the person purchasing the device is counting on it and its business partners to be trustworthy.
+	  </t>
```
Comment (WilliamDenniss) [inline draft-ietf-oauth-device-flow.xml]: The one mitigating factor here is that the user would end up on the wrong site when authorizing, and may notice. As John put it: "They would hopefully notice that they are being sent to the wrong site.". Should we include any mention of this?

File: draft-ietf-oauth-device-flow.xml [hunk 4]
```diff
@@ -636,6 +636,14 @@
           from. Thus, signals from the approving user's session and device are
           not relevant to the trustworthiness of the client device.
           </t>
+	  <t>
+	    Note that if an authorization server used with this flow is malicious,
+	    then it could man-in-the middle the backchannel flow to another authorization server.
```
Comment (WilliamDenniss) [inline draft-ietf-oauth-device-flow.xml:641]: Proposed text: 

"… to another authorization server. In this scenario, the man-in-the-middle is not completely hidden from sight as the user would end up on the authorization page on of the other server, giving them an opportunity to notice that the authorization context is wrong."

File: draft-ietf-oauth-device-flow.xml (hunk 1 above)
Comment (selfissued) [reply draft-ietf-oauth-device-flow.xml]: I'll delete the second occurrence.

Comment (selfissued) [reply draft-ietf-oauth-device-flow.xml]: My next update will be more explicit about the possibilities.

File: draft-ietf-oauth-device-flow.xml (hunk 3 above)
Comment (selfissued) [reply draft-ietf-oauth-device-flow.xml]: Ok - changed "will" to "can".

File: draft-ietf-oauth-device-flow.xml (hunk 4 above)
Comment (selfissued) [reply draft-ietf-oauth-device-flow.xml:641]: Added

**Pull request merged**